src/OpenSage.Game/Data/Utilities/ParseUtility.cs

```diff
@@ -1,11 +1,13 @@
-﻿using System.Globalization;
+﻿using System;
+using System.Globalization;
 using System.Text.RegularExpressions;
 
 namespace OpenSage.Data.Utilities
 {
     internal static class ParseUtility
     {
         private static readonly Regex FloatRegex = new Regex("^([-+]?[0-9]*\\.?[0-9]+)", RegexOptions.Compiled);
+        private static readonly Regex IntegerRegex = new Regex("^([-+]?[0-9]*)", RegexOptions.Compiled);
```

Comment:
I think this should be `^([-+]?[0-9]+)`.

Reply:
Fixed